src/engraving/dom/stafftype.h

```diff
@@ -415,6 +415,9 @@ class StaffType
     mu::PointF chordStemPosBeam(const Chord*) const;
     double   chordStemLength(const Chord*) const;
 
+    //    Check lines() == 0 to prevent conflicts when there are multiple staff;
+    bool isJianpu() const { return (name() == "Jianpu" && lines() == 0) || (xmlName() == "stdJianpu" && lines() == 0); }
```

Comment:
Wouldn't it be possible to create a new `StaffGroup::JIANPU`, so that we can implement this method more similar to `isTabStaff()` and `isDrumStaff()`? I think that would also allow us to not worry about properties that are not relevant to Jianpu like number of staff lines, so that we can just ignore the values of these properties.